javascript/src/_config/pre.js

```diff
@@ -0,0 +1,11 @@
+const fse = require("fs-extra");
+const path = require("path");
+
+const mainJsonPath = path.join(__dirname, "../_data/main.json");
+if (!fse.pathExistsSync(mainJsonPath)) {
```

Comment:
I think that we should use `fs.access` with `R_OK | W_OK` flags, like we check for file existance + if we have permissions for read/write, sure this could be extreme 😋  but yeah 🤷‍♂️

Reply:
i went for this package for simplicity, it takes care of unnecessary flags for you and it works the same across os, and about performance, we only call this line once

Reply:
Sure provides a simpler API, but I meant that we should check file permissions, imagine somehow we didn't have write permissions for your current user on that file. But as I said as well, it's not a big moto ATM so feel free to resolve this :)

Reply:
in this case, the operation will fail anyway, but yeah noted 👌🏼